tests/ts/query-examples/basic-queries/delete.ts

```diff
@@ -0,0 +1,28 @@
+import * as objection from '../../../../';
+import { lit, raw, ref, RelationMappings } from '../../../../';
+
+class Person extends objection.Model {}
+
+(async () => {
+  await Person.query().deleteById(1);
+
+  const numDeleted: number = await Person.query()
+    .delete()
+    .where(raw('lower("firstName")'), 'like', '%ennif%');
+
+  console.log(numDeleted, 'people were deleted');
```

Comment:
i don't think these will ever actually be run, will they?

Reply:
hm i guess it doesn't matter if these files are purely for giving some examples

Reply:
Yeah. For right now I'm just blindly copy-pasting from the example. I don't mind either way on these but they don't add any value, so might just remove.

Reply:
Yeah. For right now I'm just blindly copy-pasting from the example. I don't mind either way on these but they don't add any value, so might just remove.